src/StackAdmin/AzureStackAdmin/Commands.AzureStackAdmin/Nouns.cs

```diff
@@ -42,7 +42,7 @@ internal static class Nouns
         /// <summary>
         /// The noun for operations on Subscriptions as an administrator.
         /// </summary>
-        public const string TenantSubscription = Prefix + "TenantSubscription";
+        public const string TenantSubscription = Prefix + "UserSubscription";
```

Comment:
Pls Rename public const string TenantSubscription to public const string UserSubscription